### PR TITLE
fix: pci_identifier to payload data

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
To allow downloading correct PCI we need to define what pci_identifier user want's to download. Therefore I extended  request payload that is used to generate registered PCI lists to allow FE create request that is including pci_identifier required to define correct PCI model.

### CODE REVIEW

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: controller/PciManager.php [hunk 1]
```diff
@@ -227,20 +224,17 @@ public function add()
...
+        $requestPayload = $this->getPsrRequest()->getQueryParams();
         try {
-            if (!$this->hasRequestParameter('typeIdentifier')) {
-                throw new PortableElementException('Type identifier parameter missing.');
+            if (!isset($requestPayload['typeIdentifier'], $requestPayload['pciIdentifier'])) {
+                throw new PortableElementException('PCI parameter missing.');
             }
-            $identifier = $this->getRequestParameter('typeIdentifier');
-            $path = $this->getService()->export(PciModel::PCI_IDENTIFIER, $identifier);
+            $path = $this->getService()->export($requestPayload['pciIdentifier'], $requestPayload['typeIdentifier']);
```
Comment (pnal) [inline controller/PciManager.php:237]: Should `$requestPayload['pciIdentifier']` be mandatory as well?

Comment (bartlomiejmarszal) [reply controller/PciManager.php:237]: Now it's should be mandatory.

**Review by pnal — APPROVED** [review]
BE part LGTM
Nice job!
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**